demo/dh-prod.yaml

```diff
@@ -0,0 +1,409 @@
+  #    cp tls.crt tls.key $deephaven_core_directory/demo/certs/
+  #    Then, forcibly restart your local cluster
+  #    { k delete deployment dh-local || true ; } && k apply -f ./pod-dh.yaml
+  #    Now, visit https://domain.name:8433 and rejoice.
+  ports:
+    -
+      name: envoy-client
+      port: 10000
+      targetPort: 10000
+      protocol: TCP
+      nodePort: 30080
+    -
+      name: envoy-admin
+      port: 9090
+      targetPort: 9090
+      protocol: TCP
+      nodePort: 30443
+  selector:
+    # This service runs a deployment. That deployment is dh-local, below:
+    run: dh-local
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dh-local
+spec:
+  selector:
+    matchLabels:
+      run: dh-local
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: dh-local
+    spec:
+      volumes:
+        - name: envoy-conf
+          configMap:
+            # Provide the name of the ConfigMap containing the files you want
+            # to add to the container
+            name: envoy-conf
+        - name: envoy-secrets
+          secret:
+            secretName: secret-tls
+        - name: cred-cache
+          emptyDir:
+            medium: Memory
+
+      containers:
+
+        #
+        # grpc-api container.  Where all the stateful brains live
+        #
+        - name: grpc-api
+          image: us-central1-docker.pkg.dev/deephaven-oss/deephaven/grpc-api:0.0.1
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: false
+          # Change tty to true if you want to be able to shell into the box
+          tty: true
+          env:
+          - name: CONTAINER_NAME
+            value: grpc-api
+          - name: MY_NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: MY_POD_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          - name: HOST_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.hostIP
+          - name: MY_POD_SERVICE_ACCOUNT
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.serviceAccountName
+          volumeMounts:
+          - name: cred-cache
+            mountPath: /root/.ssh
+#          readinessProbe:
+#            exec:
+#              command: [ "/health/grpc_health_probe", "-addr=:8080" ]
+#            initialDelaySeconds: 2
+#            periodSeconds: 1
+#            failureThreshold: 28
+#          livenessProbe:
+#            exec:
+#              command: [ "/health/grpc_health_probe", "-addr=:8080" ]
+#            initialDelaySeconds: 10
+#            periodSeconds: 8
```

Comment:
Should the health checks be commented out?  A comment would help, if they should be.